Dockerfile.repo

```diff
@@ -32,56 +37,41 @@ RUN curl -O https://cran.rstudio.com/src/base/R-3/R-${R_VERSION}.tar.gz && \
 RUN ln -s /opt/R/${R_VERSION}/bin/R /usr/local/bin/R
 RUN ln -s /opt/R/${R_VERSION}/bin/Rscript /usr/local/bin/Rscript
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0
-ENV LD_LIBRARY_PATH $JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server/
-RUN echo "$JAVA_HOME/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
+RUN echo "$JAVA_HOME/jre/lib/$(arch)/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
```

Comment:
This seems like a duplicate of line 6. Does it serve a different purpose?

Reply:
No, this was unnecessary